src/scripts/dungeons/DungeonRunner.ts

```diff
@@ -36,11 +36,21 @@ class DungeonRunner {
         DungeonRunner.timeLeft(GameConstants.DUNGEON_TIME);
         // Dungeon size increases with each region
         let dungeonSize = GameConstants.BASE_DUNGEON_SIZE + player.region;
-        // Decrease dungeon size by 1 for every 10, 100, 1000 etc completes
-        dungeonSize -= Math.max(0, App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]().toString().length - 1);
-        const flash = App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]() >= 200;
+
+
         // Dungeon size minimum of MIN_DUNGEON_SIZE
-        DungeonRunner.map = new DungeonMap(Math.max(GameConstants.MIN_DUNGEON_SIZE, dungeonSize), flash);
+        if (!App.game.challenges.list.hardDungeons.active()) {
+            const flash = App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]() >= 200;
+            // Decrease dungeon size by 1 for every 10, 100, 1000 etc completes
+            dungeonSize -= Math.max(0, App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]().toString().length - 1);
+            DungeonRunner.map = new DungeonMap(Math.max(GameConstants.MIN_DUNGEON_SIZE, dungeonSize), flash);
+        } else {
+            const flash = false;
+            DungeonRunner.map = new DungeonMap(Math.max(GameConstants.MIN_DUNGEON_SIZE, dungeonSize), flash);
+        }
+
+
+
```

Comment:
Might be worth simplifying this a little bit and removing some duplication
```suggestion
		let flash = false;

        if (!App.game.challenges.list.hardDungeons.active()) {
        	// Reveal dungeon tiles nearby player
            const flash = App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]() >= 200;
            // Decrease dungeon size by 1 for every 10, 100, 1000 etc completes
            dungeonSize -= Math.max(0, App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]().toString().length - 1);
        }

	    DungeonRunner.map = new DungeonMap(Math.max(GameConstants.MIN_DUNGEON_SIZE, dungeonSize), flash);
```